### PR TITLE
convert: add pt_BR translation

### PR DESCRIPTION
**Pull request by xRuiAlves**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/common/convert.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# convert
+
+> Ferramenta de converção de imagens da Imagemagick.
```
Comment (waldyrious) [inline pages.pt_BR/common/convert.md]:
```suggestion
> Ferramenta de conversão de imagens da Imagemagick.
```

File: pages.pt_BR/common/convert.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Escalar uma imagem, mantendo as suas proporções originais, para uma dimensão máxima de 640x480:
+
+`convert {{imagem.png}} -resize 640x480 {{nova_imagem.png}}`
+
+- Juntar várias imagens horizontalmente:
+
+`convert {{imagem1.png}} {{imagem2.png}} {{imagem3.png}} +append {{nova_imagem.png}}`
+
+- Criar um GIF a partir de uma série de imagens, com um atraso de 100ms entre elas:
```
Comment (waldyrious) [inline pages.pt_BR/common/convert.md]: Talvez "intervalo" funcione melhor do que "atraso", o que achas?

File: pages.pt_BR/common/convert.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`convert {{imagem1.png}} {{imagem2.png}} {{imagem3.png}} -delay {{100}} {{nova_imagem.gif}}`
+
+- Criar uma nova imagem com um fundo sólido apenas:
```
Comment (waldyrious) [inline pages.pt_BR/common/convert.md]: Eu mencionaria na descrição a cor do fundo, até porque é especificada no comando.

**Comment by xRuiAlves** [comment]
@waldyrious Implemented requested changes :smiley: :+1:

File: pages.pt_BR/common/convert.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# convert
+
+> Ferramenta de conversão de imagens da Imagemagick.
+> Mais informação: <https://imagemagick.org/script/convert.php>.
```
Comment (schneiderl) [inline pages.pt_BR/common/convert.md]:
```suggestion
> Mais informações: <https://imagemagick.org/script/convert.php>.
```

**Comment by xRuiAlves** [comment]
@schneiderl Implemented requested changes :+1:

**Pull request merged**